<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logManagerDlg</class>
 <widget class="QDialog" name="logManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="log_tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>159</y>
     <width>1001</width>
     <height>601</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>操作员</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>描述</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>91</y>
     <width>76</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
    </font>
   </property>
   <property name="text">
    <string>开始时间:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit1">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>87</y>
     <width>150</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QDateTimeEdit
{
	border-color: rgba(255, 255, 255, 0);
	background-color: rgb(89, 107, 169);	
 	border:1px;
	font-size:17px;
    border-right:0px;
	border-radius:4px;
	color:white;
selection-background-color:rgb(136,150,199);
}


QDateTimeEdit::down-arrow
{
	image: url(:/png/date_sel.png);
}

QDateTimeEdit::down-arrow:hover
{
	image: url(:/png/date_sel.png);
}

QDateTimeEdit::drop-down{
	border:none;	
    background:transparent;
	padding-right:2px;
	width:30px;
	height:30px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>87</y>
     <width>91</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateTimeEdit
{
	color:white;
	background-color: rgb(89, 107, 169);	
    border:0px solid;
	border-radius:4px;
	border-right:0px;
}

QDateTimeEdit::up-button
{
/*subcontrol-origin: margin;  */
    subcontrol-position: top right; 

	image: url(:/images/up.png);

}


QDateTimeEdit::down-button
{
/*subcontrol-origin: margin;  */
    subcontrol-position: bottom right; 
margin-top:-1px;
	image: url(:/images/down.png);
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>92</y>
     <width>76</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
    </font>
   </property>
   <property name="text">
    <string>结束时间:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit2">
   <property name="geometry">
    <rect>
     <x>467</x>
     <y>87</y>
     <width>150</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QDateTimeEdit
{
	border-color: rgba(255, 255, 255, 0);
	background-color: rgb(89, 107, 169);	
 	border:1px;
	font-size:17px;
    border-right:0px;
	border-radius:4px;
	color:white;
selection-background-color:rgb(136,150,199);
}


QDateTimeEdit::down-arrow
{
	image: url(:/png/date_sel.png);
}

QDateTimeEdit::down-arrow:hover
{
	image: url(:/png/date_sel.png);
}

QDateTimeEdit::drop-down{
	border:none;	
    background:transparent;
	padding-right:2px;
	width:30px;
	height:30px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit2">
   <property name="geometry">
    <rect>
     <x>627</x>
     <y>87</y>
     <width>91</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateTimeEdit
{
	color:white;
	background-color: rgb(89, 107, 169);	
    border:0px solid;
	border-radius:4px;
	border-right:0px;
/*margin-right:6px;*/
}

QDateTimeEdit::up-button
{
/*subcontrol-origin: margin;  */
    subcontrol-position: top right; 

	image: url(:/images/up.png);
}


QDateTimeEdit::down-button
{
/*subcontrol-origin: margin;  */
    subcontrol-position: bottom right; 
margin-top:-1px;
	image: url(:/images/down.png);
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchBtn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>86</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
   <property name="icon">
    <iconset resource="fastvideo.qrc">
     <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="outputBtn">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>86</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
   <property name="icon">
    <iconset resource="fastvideo.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pastPage">
   <property name="geometry">
    <rect>
     <x>944</x>
     <y>125</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="fastvideo.qrc">
     <normaloff>:/images/prev.png</normaloff>:/images/prev.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPage">
   <property name="geometry">
    <rect>
     <x>982</x>
     <y>125</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="fastvideo.qrc">
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="PageLabel">
   <property name="geometry">
    <rect>
     <x>894</x>
     <y>130</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0/0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="taskLogBtn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>141</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>任务日志</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="fastvideo.qrc"/>
 </resources>
 <connections/>
</ui>
